### ISSUE TITLE
Как внести вклад в развитие проекта ? 

### ISSUE THREAD

**Issue by gromdron** [LineyKin/mysocnet#1]
Как развернуть данный проект ? sql и миграций нет, установщика нет. Как предполагается система open source развитие проекта, если его нельзя установить ? И хотелось бы увидеть описание ;)

**Comment by LineyKin** [comment]
Прежде всего благодарю за ваши поправки)
Но у меня проект учебный, на нём я просто оттачиваю вёрстку и написание основных атрибутов сайтов: регистрация, авторизация, подписка на другого пользователя, удаление страницы, настройки своей страницы(аватар, фон, информация о себе). Ну и позднее решил пришить к сайту Facemash (спасибо фильму "социальная сеть" и лично Марку Цукербергу). В этом, так сказать, конкурсе участвуют все пользователи с аватаром. Ну а выбирать могут и незарегистрированные. В планах прописать обмен сообщениями. 
На гитхаб я выложил по двум причинам: 1 - мне надо было начать осваивать гит и гитхаб. 2 - я кидаю ссылки на гитхаб потенциальным работодателям, это удобно весьма.
Многое до сих пор не знаю, про миграции и установщиков в том числе.

**Comment by LineyKin** [comment]
Забавно, ты подписан на человека, с которым я год назад пересекался на собеседовании в их appatbox)

**Comment by gromdron** [comment]
1) Очень отрадно, что идет освоение гитхаба и учебные проекты, я очень рад, респект.
2) Пусть проект и учебный, но ссылку на него в резюме указывать не стоит (слишком сырой, мы смотрели). После прочтения резюме, вызывает даже немного негативные эмоции.
3) Я тут чтобы "отдохнуть от работы" и "вспомнить гит", а так как у тебя учебный проект и открытый проект, то тоже повод внести свою лепту.

**Comment by LineyKin** [comment]
Да, мне и самому приходила мысль, что ссылки на гитхаб мало, но чтоб это вызывало негатив и подумать не мог. Лучше кидать сразу рабочие ссылки?
а по поводу "внести свою лепту" - милости просим, мне же полезней) Как освободится время - поиграюсь у себя с твоими исправлениями)
